README.md

```diff
@@ -39,7 +39,13 @@ This builds the app for production to the `build` folder.
 
 It correctly bundles React in production mode and optimizes the build for the best performance. The build is minified and the filenames include the hashes. Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Note: for this project, we use a node server (see `scripts/server.js`).
+
+### `yarn dev`
+
+This runs the app in the development mode, with the [-dev AMO API](https://addons-server.readthedocs.io/).
+
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser. The page will reload if you make edits. You will also see any lint errors in the console.
 
 ### `yarn eject`
```

Comment:
Yes, it is a command that can be run by a developer so it should be documented. Also, it can be useful to temporarily eject (then revert) if you want to figure out how something is configured. 

The only commands that I think are not worth documenting are those run by machines (such as `yarn test-ci`). There is a note in the readme about undocumented commands.